rllib/offline/offline_prelearner.py

```diff
@@ -86,8 +86,8 @@ def __init__(
         self,
         config: "AlgorithmConfig",
         learner: Union[Learner, list[ActorHandle]],
+        locality_hints: List[str] = None,
```

Comment:
nit: re-add `Optional`, b/c it could be None (and the default is actually None).

Reply:
Great catch. Had to add this here b/c the master has not yet the changes from streaming optimization branch in it - there the locality hints are gone.